client/rpc/src/overrides/schema_v1_override.rs

```diff
@@ -1,31 +1,35 @@
-// Copyright 2017-2020 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
 // This file is part of Frontier.
-
-// Substrate is free software: you can redistribute it and/or modify
+//
+// Copyright (c) 2020 Parity Technologies (UK) Ltd.
```

Comment:
Same here. If this file copied anything from Substrate repo (which it seems to indicate so by having had a Substrate copyright header), then year from 2017 should be kept.